osu.Game/Screens/Menu/SystemTitle.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Framework.Input.Events;
+using osu.Framework.Platform;
+using osu.Game.Online.API.Requests;
+using osu.Game.Online.API.Requests.Responses;
+
+namespace osu.Game.Screens.Menu
+{
+    public partial class SystemTitle : CompositeDrawable
+    {
+        internal Bindable<APISystemTitle?> Current { get; } = new Bindable<APISystemTitle?>();
+
+        private Container content = null!;
+        private CancellationTokenSource? cancellationTokenSource;
+        private SystemTitleImage? currentImage;
+
+        [BackgroundDependencyLoader]
+        private void load(GameHost? gameHost)
+        {
+            Anchor = Anchor.BottomCentre;
+            Origin = Anchor.BottomCentre;
+            AutoSizeAxes = Axes.Both;
+
+            InternalChild = content = new ClickableContainer
+            {
+                AutoSizeAxes = Axes.Both,
+                Action = () =>
+                {
+                    if (!string.IsNullOrEmpty(Current.Value?.Url))
+                        gameHost?.OpenUrlExternally(Current.Value.Url);
+                }
+            };
+        }
+
+        protected override bool OnHover(HoverEvent e)
+        {
+            content.ScaleTo(1.1f, 500, Easing.OutBounce);
+            return base.OnHover(e);
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            content.ScaleTo(1f, 500, Easing.OutBounce);
+            base.OnHoverLost(e);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Current.BindValueChanged(_ => loadNewImage(), true);
+
+            checkForUpdates();
+            Scheduler.AddDelayed(checkForUpdates, TimeSpan.FromMinutes(15).TotalMilliseconds, true);
```

Comment:
Not sure if this polling is necessary. Probably can be removed if just a once-off call at startup is deemed acceptable.

Reply:
Refreshing is best. It will already only attempt a refresh when returning to the main menu, which should be pretty fine. I'd almost say to just check each time we return, but I guess a bit of delay is worth having.